bundles/org.smarthomej.binding.viessmann/src/main/java/org/smarthomej/binding/viessmann/internal/dto/ThingMessageDTO.java

```diff
@@ -0,0 +1,139 @@
+/**
+ * Copyright (c) 2021 Contributors to the SmartHome/J project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.smarthomej.binding.viessmann.internal.dto;
+
+import org.smarthomej.binding.viessmann.internal.dto.features.FeatureCommands;
+
+/**
+ * Superclass for all Thing message types.
+ * 
+ * @author Ronny Grun - Initial contribution
+ */
+public class ThingMessageDTO {
+
+    private String type;
+
+    private String channelType;
+
+    private String uom;
+
+    private String value;
+
```

Comment:
```suggestion
    private String type;
    private String channelType;
    private String uom;
    private String value;
```

remove empty lines after class opening and between fields (if not needed for grouping).